emojivoto/README.md

```diff
@@ -75,3 +75,52 @@ If you'd like to run the bot manually:
 export WEB_HOST=localhost:8080 # replace with your web location
 go run emojivoto-web/cmd/vote-bot/main.go
 ```
+
+## Local Development
+
+### Emojivoto webapp
+
+This app is written with React and bundled with webpack.
+Use the following to run the emojivoto go services and develop on the frontend.
```

Comment:
add a note about `dep ensure`? related: running `dep ensure` modified `Gopkg.lock` for me.

also, protoc?

... now that i'm thinking about it, maybe just `make build` to cover all this? (and note this requires docker?)

Reply:
hmmmm you mean like adding another rule in the make file? or doing a docker-compose up, stopping the web service and running with webpack dev server? mostly I'm just trying to document how one would run this if one wanted to also have webpack-dev-server.

Reply:
gonna merge this, we can clarify in a follow-up.